pages/income.jsx

```diff
@@ -1,14 +1,15 @@
 import { Link } from "react-router-dom";
 
 const Expense = () => {
-    return(
-        <div className="expense">
-
-<Link to="/" className="start">Home</Link>
-
-            <h2 className="text">This is the Expense Tracker Site!</h2>
-        </div>
-    );
-}
-
-export default Expense;
\ No newline at end of file
+  return (
+    <div className="expense">
+      <Link to="/" className="start">
+        Startseite
+      </Link>
+
+      <h2 className="text">This is the Expense Tracker Site!</h2>
+    </div>
+  );
+};
+
```

Comment:
This is just an empty site with no value for the user. Would remove it from the pull request and put it back in another story.